extensions/sql-database-projects/README.md

```diff
@@ -17,10 +17,10 @@ Please report issues and feature requests [here.](https://github.com/microsoft/a
 
 ## Getting Started with Database Projects
 
-* Create a new database project by going to the **Projects** viewlet or by searching **Projects: New** in the command palette.
-* Existing database projects can be opened by going to the **Projects** viewlet or by searching **Projects: Open Existing** in the command palette.
+* Create a new database project by going to the **Database Projects** view or by searching **Database Projects: New** in the command palette.
```

Comment:
Do you think `code blocks` rather than **bold blocks** would make the exact strings more visible in the readme?

Reply:
good idea! Updated them